doc/api/http.md

```diff
@@ -2838,6 +2838,10 @@ Found'`.
 <!-- YAML
 added: v0.1.13
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41310
+    description: The `noDelay`, `keepAlive` and `keepAliveInitialDelay`
+                 options are supported now.
```

Comment:
I think this PR only changes `noDelay` now? (Same with the change to net.md).

Reply:
Technically yes. But in the PR where I originally added it I forgot to update the docs' changelog. 😞

Shall I revert the modification or is it good?

Reply:
Ah I didn't spot the PR-URL was for an earlier PR. Keep those, but add a new entry for this PR noting the default has been changed for `noDelay.

Reply:
Done! Does it look fine now?